raindrops/manferna/raindrops.go

```diff
@@ -0,0 +1,37 @@
+package raindrops
+
+import (
+	"strconv"
+)
+
+func Convert(number int) string {
+
+	const threeAsFactor = "Pling"
+	const fiveAsFactor = "Plang"
+	const sevenAsFactor = "Plong"
+
+	var raindropsResult = ""
+
+	// potencial factor 3
+	if number%3 == 0 {
+		raindropsResult = raindropsResult + threeAsFactor
+	}
+
+	//potencial factor 5
+	if number%5 == 0 {
+		raindropsResult = raindropsResult + fiveAsFactor
+	}
+
+	//potencial factor 7
+	if number%7 == 0 {
+		raindropsResult = raindropsResult + sevenAsFactor
+	}
+
+	if raindropsResult == "" {
+		numberToString := strconv.Itoa(number)
+		return numberToString
+	} else {
```

Comment:
No, just for me it's easier to read it. But it can be removed.

Reply:
Sure, I understand, it's a style and a personal thing. Usually people will mention that on MR and even though they won't block and MR because of it, it just generates noise. And if you eventually decide to change this (not here, but on some work code), that'll also mean more load on the CI system.